app/src/components/species/components/SpeciesCard.tsx

```diff
@@ -1,23 +1,44 @@
-import { Box, Typography } from '@mui/material';
+import { Chip, Stack, Typography } from '@mui/material';
 
 interface ISpeciesCard {
   name: string;
-  subtext: string;
+  scientificName: string;
+  tsn: number;
 }
 const SpeciesCard = (props: ISpeciesCard) => {
   return (
-    <Box>
-      <Box>
-        <Typography variant="subtitle1" fontWeight="bold">
-          {props.name}
-        </Typography>
-      </Box>
-      <Box my={0.25}>
-        <Typography variant="subtitle2" color="textSecondary">
-          {props.subtext}
-        </Typography>
-      </Box>
-    </Box>
+    <Stack flexDirection="row" justifyContent="space-between" width="100%">
+      <Typography component="span" variant="body1">
+        {props.name ? (
+          <>
+            <Typography
+              component="strong"
+              sx={{
+                display: 'inline-block',
+                fontWeight: 700,
+                '&::first-letter': {
+                  textTransform: 'capitalize'
+                }
+              }}>
+              {props.name}
+            </Typography>
+            &nbsp;(<em>{props.scientificName}</em>)
+          </>
+        ) : (
+          <em>{props.scientificName}</em>
+        )}
+      </Typography>
+      <Chip
+        title="Taxonomic serial number"
+        label={'TSN:' + props.tsn}
```

Comment:
Nit: use template string. `props.tsn` is probably type `number`, and `string + number` is very unpredictable in JS.

Reply:
Will fix via another branch.